mediawiki/stack.yml

```diff
@@ -0,0 +1,21 @@
+# MediaWiki with MariaDB
+#
+# Access via "http://localhost:8080"
+#   (or "http://$(docker-machine ip):8080" if using docker-machine)
+version: '3'
+services:
+  mediawiki:
+    image: mediawiki
+    ports:
+      - 8080:80
+    links:
+      - database
+    volumes:
+      - /var/www/html/images
+  database:
+    image: mariadb
+    environment:
+      # @see https://phabricator.wikimedia.org/source/mediawiki/browse/master/includes/DefaultSettings.php
+      MYSQL_DATABASE: 'my_wiki'
+      MYSQL_USER: 'wikiuser'
```

Comment:
If the users are meant to use the `wikiuser`, then the password env is required.  `MYSQL_USER` will not be created, since there is no `MYSQL_PASSWORD`.